### PR TITLE
chore: add ingest integration tests for adcs information

### PR DESCRIPTION
**Pull request by urangel**
## Description
This is changeset includes a suite of ingest integration tests for data collected for ADCS.

## Motivation and Context
We lack coverage in this area so these tests should outline expected behavior.

## Types of changes

-   [x] Chore (a change that does not modify the application functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [ ] Documentation updates are needed, and have been made accordingly.
-   [x] I have added and/or updated tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] My changes include a database migration.

### CODE REVIEW

File: packages/cue/bh/ad/ad.cue [hunk 1]
```diff
@@ -384,6 +384,13 @@ ApplicationPolicies: types.#StringEnum & {
 	representation: "applicationpolicies"
 }
 
+IssuancePolicies: types.#StringEnum & {
```
Comment (rvazarkar) [inline packages/cue/bh/ad/ad.cue:387]: Do these new properties have any impact on esc1?

Comment (urangel) [reply packages/cue/bh/ad/ad.cue:387]: These will get ingested regardless but having them defined allows us to display them better in the entity info

**Review by rvazarkar — APPROVED** [review]

**Comment by rvazarkar** [comment]
This looks great, thank you for putting together all these tests

**Review by zinic — APPROVED** [review]
:+1:

**Pull request merged**